NOTICE

```diff
@@ -13,18 +13,3 @@ Declared Project License
 ------------------------
 The artifacts of this project are made available under the terms of the MIT License.
 Please refer to the LICENSE file for more information.
-
-
-Third-party Work in this Repository
------------------------------------
-This project includes XSD and JSON Schema files, developed at
-Plattform Industrie 4.0, Bundesministerium für Wirtschaft und Energie (https://www.plattform-i40.de)
-
-Published in "Details of the Asset Administration Shell – Part 1"
-at https://www.plattform-i40.de/PI40/Redaktion/DE/Downloads/Publikation/Details-of-the-Asset-Administration-Shell-Part1.html
-
-under the terms of the Creative Commons Attribution-NoDerivs 3.0 Germany License (CC BY-ND 3.0 DE)
-https://creativecommons.org/licenses/by-nd/3.0/de/deed.en
-
-These files are located at `basyx/aas/adapter/json/aasJSONSchema.json` and
-`basyx/aas/adapter/xml/{AAS,AAS_ABAC,IEC61360}.xsd`.
```

Comment:
Is it correct to entirely remove the information about the licenses of external files from this repository? I'm not an expert on this, but I think removing the notices effectively relicenses these files under our MIT license, which isn't right.

Reply:
Good idea! Currently, the only third party files the SDK contains are the schema files. We also package them as part of the SDK in `setup.py`, as they are required by the compliance tool.

We're planning on moving the compliance tool to its own package anyway, so we can remove the files from our repo entirely once we do this.

Reply:
Alright, let's just postpone this PR until we do. I'll create an issue.